collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadList.java

```diff
@@ -409,6 +410,14 @@ public void onClick(DialogInterface dialog, int which) {
                 alertShowing = false;
 
                 return new AuthDialogUtility().createDialog(this, this, null);
+            case CANCEL_DIALOG:
+                cancelDialog = new ProgressDialog(this);
+                cancelDialog.setTitle(getString(R.string.please_wait));
```

Comment:
You might consider having the title be “Canceling” and have the message be please_wait.